src/main/java/killjoy/main/KillJoy.java

```diff
@@ -0,0 +1,134 @@
+package killjoy.main;
+
+import killjoy.task.Deadline;
+import killjoy.task.Event;
+import killjoy.task.Task;
+import killjoy.task.Todo;
+import killjoy.processing.ProcessTasks;
+import killjoy.processing.Storage;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Represents the main class of the KillJoy application.
+ * Contains the main method to run the application.
+ */
+public class KillJoy {
+    private ProcessTasks processTasks;
+    private Storage saveAndLoad;
+    private UserInterface ui;
+    private ArrayList<Task> taskList;
+    private int taskCount = 0;
+
+    public KillJoy() {
+        this.ui = new UserInterface(this);
+        this.processTasks = new ProcessTasks(this, ui);
+        this.saveAndLoad = new Storage(this, processTasks);
+        this.taskList = new ArrayList<>();
+    }
+
+    public int getTaskCount() {
+        return taskCount;
+    }
+
+    /**
+     * Increases the task count by 1.
+     */
+    public void increaseTaskCount() {
+        this.taskCount++;
+    }
+
+    /**
+     * Decreases the task count by 1.
+     */
+    public void decreaseTaskCount() {
+        this.taskCount--;
+    }
+
+    /**
+     * Adds a task to the task list.
```

Comment:
There should be an empty line between the description and the parameter section. The same goes for the rest of the javadocs :)